rskj-core/src/main/java/org/ethereum/core/BlockFactory.java

```diff
@@ -118,53 +140,54 @@ public BlockHeader decodeHeader(byte[] encoded, boolean compressed) {
     }
 
     private BlockHeader decodeHeader(RLPList rlpHeader, boolean compressed, boolean sealed) {
-        byte[] parentHash = rlpHeader.get(0).getRLPData();
-        byte[] unclesHash = rlpHeader.get(1).getRLPData();
-        byte[] coinBaseBytes = rlpHeader.get(2).getRLPData();
+        byte[] parentHash = rlpHeader.get(PARENT_HASH_INDEX).getRLPData();
+        byte[] unclesHash = rlpHeader.get(UNCLES_HASH_INDEX).getRLPData();
+        byte[] coinBaseBytes = rlpHeader.get(COINBASE_INDEX).getRLPData();
         RskAddress coinbase = RLP.parseRskAddress(coinBaseBytes);
-        byte[] stateRoot = rlpHeader.get(NUMBER_OF_EXTRA_HEADER_FIELDS).getRLPData();
+        byte[] stateRoot = rlpHeader.get(STATE_ROOT_INDEX).getRLPData();
         if (stateRoot == null) {
             stateRoot = EMPTY_TRIE_HASH;
         }
 
-        byte[] txTrieRoot = rlpHeader.get(4).getRLPData();
+        byte[] txTrieRoot = rlpHeader.get(TX_TRIE_ROOT_INDEX).getRLPData();
         if (txTrieRoot == null) {
             txTrieRoot = EMPTY_TRIE_HASH;
         }
 
-        byte[] receiptTrieRoot = rlpHeader.get(5).getRLPData();
+        byte[] receiptTrieRoot = rlpHeader.get(RECEIPT_TRIE_ROOT_INDEX).getRLPData();
         if (receiptTrieRoot == null) {
             receiptTrieRoot = EMPTY_TRIE_HASH;
         }
 
-        byte[] extensionData = rlpHeader.get(6).getRLPData(); // rskip351: logs bloom when decoding extended, list(version, hash(extension)) when compressed
-        byte[] difficultyBytes = rlpHeader.get(7).getRLPData();
+        byte[] extensionData = rlpHeader.get(EXTENSION_DATA_INDEX).getRLPData(); // rskip351: logs bloom when decoding
+        // extended, list(version,
+        // hash(extension)) when compressed
```

Comment:
Sorry to be nitpicking at this point, but can we fix the formatting of this comment?